shared/domain/src/main/java/gov/va/vro/model/mas/MasAutomatedClaimPayload.java

```diff
@@ -46,6 +47,8 @@ public class MasAutomatedClaimPayload implements Auditable {
 
   @NotNull @Valid private ClaimDetail claimDetail;
 
+  @Builder.Default @NotNull private String idType = CLAIM_V2_ID_TYPE;
+
```

Comment:
This comment is for line 44 above. It should be:
```
   private String collectionId;
```
See my comment for [MasIntegrationProcessors.java](https://github.com/department-of-veterans-affairs/abd-vro/pull/1112/files#diff-af0561bfcedd9b27f11cfb67d867cf4ffc9f7a4685f4848fe2483986f3d31d89) line 91.

Reply:
see earlier comment re: risk to timelines while not disagreeing. This PR didn't alter that field and I don't believe at this point it should.